.github/actions/nf-test/action.yml

```diff
@@ -54,10 +54,20 @@ runs:
         conda-solver: libmamba
         conda-remove-defaults: true
 
+    # Set up secrets
+    - name: Set up Nextflow secrets
+      if: env.SENTIEON_ENCRYPTION_KEY != '' && env.SENTIEON_LICENSE_MESSAGE != ''
+      shell: bash
+      run: |
+        python -m pip install cryptography
+        nextflow secrets set SENTIEON_AUTH_DATA $(python3 bin/license_message.py encrypt --key "$SENTIEON_ENCRYPTION_KEY" --message "$SENTIEON_LICENSE_MESSAGE")
```

Comment:
Is `license_message.py` only used for CI? If so, could we store it in .github so folks don't confuse it for something they need to care about?

Reply:
sure.